### PR TITLE
More variable naming conventions

### PR DESCRIPTION
**Pull request by rauhs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1226,12 +1226,16 @@ hints for the pairwise grouping with comments or empty lines.
     * in functions:
         * `f`, `g`, `h` - function input
         * `n` - integer input usually a size
-        * `index` - integer index
+        * `index`, `i` - integer index
         * `x`, `y` - numbers
+        * `xs` - sequence
+        * `m` - map
         * `s` - string input
+        * `re` - regular expression
```
Comment (bbatsov) [inline README.md]: Having seen this one much. `rx` and `regex` might be more popular?

Comment (arrdem) [inline README.md]: core uses re for the regex built in functions

**Comment by bbatsov** [comment]
:+1: Thanks!

File: README.md (hunk 1 above)
Comment (bbatsov) [inline README.md]: Totally missed this, but shouldn't be surprised given the names of the functions themselves. :-)

**Pull request merged**